<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlDictionary</class>
 <widget class="QWidget" name="ControlDictionary">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dicionários</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="dictionaries">
       <property name="title">
        <string>Dicionários</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <widget class="QToolButton" name="addDict">
          <property name="toolTip">
           <string>Adicionar dicionário.</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../Application/resource.qrc">
            <normaloff>:/icons/add24x24.png</normaloff>:/icons/add24x24.png</iconset>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="4">
         <widget class="QTableWidget" name="listDict">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Dicionário</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="3">
         <widget class="QToolButton" name="rmDict">
          <property name="toolTip">
           <string>Remover dicionário.</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../Application/resource.qrc">
            <normaloff>:/icons/remove24x24.png</normaloff>:/icons/remove24x24.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QLineEdit" name="nameDict"/>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nome</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QGroupBox" name="words">
       <property name="title">
        <string>Palavras</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="4" column="0" colspan="3">
         <widget class="QTableWidget" name="entries">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Código</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Palavra</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="3" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QToolButton" name="addEntry">
          <property name="toolTip">
           <string>Adicionar palavra.</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../Application/resource.qrc">
            <normaloff>:/icons/add24x24.png</normaloff>:/icons/add24x24.png</iconset>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QToolButton" name="rmEntry">
          <property name="toolTip">
           <string>Remover palavra.</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../Application/resource.qrc">
            <normaloff>:/icons/remove24x24.png</normaloff>:/icons/remove24x24.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Código</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Palavra</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="code"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="word"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Application/resource.qrc"/>
  <include location="../Application/resource.qrc"/>
 </resources>
 <connections/>
</ui>
